dftimewolf/lib/state.py

```diff
@@ -0,0 +1,73 @@
+"""This class maintains the internal dfTimewolf state.
+
+Use it to track errors, abort on global failures, cleanup after modules, etc.
+"""
+
+import sys
+
+class DFTimewolfState(object):
+  """The main State class.
+
+  Attributes:
+    errors: [(str, bool)] The errors generated by a module. These should be
```

Comment:
The type hinting in the docstrings are inconsistent with the rest of the codebase. Maybe put in a TODO to refactor the rest as well (in a later PR)

Reply:
Will do.